opentracing-impl-java8/src/main/java/io/opentracing/AbstractSpanBuilder.java

```diff
@@ -11,7 +11,7 @@
  * or implied. See the License for the specific language governing permissions and limitations under
  * the License.
  */
-package io.opentracing.propagation;
+package io.opentracing;
```

Comment:
(somehow UnIntelliJ moved a bunch of stuff in my previous PR; these moves back are not related to the PR but should happen anyway.)